CONTRIBUTING.md

```diff
@@ -41,51 +39,18 @@ We are using priority labels from **P0** to **P4** for our issues. So if you are
 * P4: Low  - Fix this when you don't see any issues with the other priorities.
 
 # How to contribute documentation
+The Open Build Service documentation is hosted in a separated repository called [obs-docu](https://github.com/openSUSE/obs-docu). Please send pull-requests against this repository. 
 
-The Open Build Service documentation is hosted in a separated repository available on [GitHub](https://github.com/openSUSE/obs-docu). How you can contribute to our documentation is described on our [project page](http://openbuildservice.org/help/manuals/obs-reference-guide/appendix.work_on_obs_book.html). Needless to say that contributions are highly welcome, right?;-)
-
-# Conduct
+# How to conduct yourself when contributing
```

Comment:
It's just a duplication of what is written in the README. Why send people through hoops?